base/src/lib/__snapshots__/export-schema.test.ts.snap

```diff
@@ -53,3 +53,35 @@ exports[`export-schema > typescript > should return typescript definition [inlin
 
 "
 `;
+
+exports[`export-schema > typescript > should treat string fields with enums as TS string unions 1`] = `
+"interface Test {
+  _id: string
+  address: {
+    country: {
+      code?: string
+      code?: string
+      title: string
+      title: string
+    }
+    line1: string
+    line1: string
+  }
+  age?: number
+  alive: boolean
+  connection?: string
+  created_at: Date
+  friends_contacts: {
+    email: string
+    email: string
+    name: string
+    name: string
+  }[]
+  name: string
+  name: string
+  tags?: string[]
+  gender?: male | female
```

Comment:
This should be

```suggestion
  gender?: 'male' | 'female'
```

Reply:
I would assume adding the `gender` definition will affect the other test snapshots but it isn't. I don't know if those changes were intentionally removed.

But you won't need to write a whole new test. Just adding the schema definition for an enum is fine. The other tests will capture it.

If I'm unclear, text me in the Discord